docker-compose.yaml

```diff
@@ -62,7 +62,7 @@ services:
     volumes:
       - .:/dockermount
     ports:
-      - 8000:8000
+      - 8500:8000
```

Comment:
Broker backend runs their DB at port 8000, making it impossible to run both at the same time. For future ease, I would advise setting our ports between 8500 and 8799, as broker tends to be all over the place with theirs but avoid that less popular range

Reply:
This isn't a terrible idea. I would suggest that instead of modifying it in this PR we discuss it during a dev meeting to see if USAspending should change or Broker should change.

Reply:
A+ for initiative.  Easier to ask for forgiveness...  Plus it got the conversation started.